modules/ocf_tv/files/i3/config

```diff
@@ -9,9 +9,14 @@
 #
 # Please see http://i3wm.org/docs/userguide.html for a complete reference!
 
-exec feh --bg-scale '/opt/share/background.png'
 
-set $mod Mod1
+set $mod        Mod4
```

Comment:
This could be a bad idea, since we don't want the super key to conflict from inside vncviewer